app/include/zmk/events/mouse_button_state_changed.h

```diff
@@ -19,8 +19,8 @@ struct zmk_mouse_button_state_changed {
 
 ZMK_EVENT_DECLARE(zmk_mouse_button_state_changed);
 
-static inline struct zmk_mouse_button_state_changed_event *
-zmk_mouse_button_state_changed_from_encoded(uint32_t encoded, bool pressed, int64_t timestamp) {
-    return new_zmk_mouse_button_state_changed((struct zmk_mouse_button_state_changed){
+static inline int raise_zmk_mouse_button_state_changed_from_encoded(uint32_t encoded, bool pressed,
```

Comment:
Should we add a `raise_zmk_keycode_state_changed_from_encoded()` function too?

Reply:
Indeed. Added and refactored to use.